### PR TITLE
Feature/user sign up

### PR DESCRIPTION
**Pull request by kencruz07**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      render json: {message: 'User created successfully'}
```
Comment (penakamaayo) [inline app/controllers/users_controller.rb]: @kencruz07 Use =>

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    end
+  end
+
+  def user_params
```
Comment (penakamaayo) [inline app/controllers/users_controller.rb:16]: @kencruz07  make this private

**Comment by kencruz07** [comment]
@penakamaayo Please review. Thanks. Haha.

File: config/routes.rb [hunk 3]
```diff
@@ -14,6 +14,8 @@
   # Example resource route (maps HTTP verbs to controller actions automatically):
   #   resources :products
 
+  resources :users
```
Comment (penakamaayo) [inline config/routes.rb]: @kencruz07  do we need all actions on this?

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      render json: {:message => 'User created successfully'}
```
Comment (penakamaayo) [inline app/controllers/users_controller.rb]: @kencruz07  :json => {...} haha

**Comment by kencruz07** [comment]
@penakamaayo Okay na ata haha. Thanks.

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      render :json => {:message => 'User created successfully'}
+    end
+  end
+
+
+
```
Comment (penakamaayo) [inline app/controllers/users_controller.rb:11]: @kencruz07  last. 5 line space haha

**Comment by kencruz07** [comment]
@penakamaayo Sorry haha eto na :))

**Pull request merged**